src/Parser/Core/Modules/Items/FoodChecker.js

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+
+import Analyzer from 'Parser/Core/Analyzer';
+import SUGGESTION_IMPORTANCE from 'Parser/Core/ISSUE_IMPORTANCE';
+import SPELLS from 'common/SPELLS';
+
+const LOWER_FOOD_IDS = [
+  // very low
+  SPELLS.KUL_TIRAMISU.id,
+  SPELLS.LOA_LEAF.id,
+  SPELLS.RAVENBERRY_TARTS.id,
+  SPELLS.MON_DAZI.id,
+  // somwhat low
+  SPELLS.HONEY_GLAZED_HAUNCHES.id,
+  SPELLS.SAILOR_PIE.id,
+  SPELLS.SWAMP_FISH_N_CHIPS.id,
+  SPELLS.SPICED_SNAPPER.id,
+  // low
+  SPELLS.GALLEY_BANQUET_AGI.id,
+  SPELLS.GALLEY_BANQUET_INT.id,
+  SPELLS.GALLEY_BANQUET_STR.id,
+  SPELLS.GALLEY_BANQUET_STA.id,
+];
+
+const HIGHER_FOOD_IDS = [
+  SPELLS.BOUNTIFUL_CAPTAIN_FEAST_AGI.id,
+  SPELLS.BOUNTIFUL_CAPTAIN_FEAST_INT.id,
+  SPELLS.BOUNTIFUL_CAPTAIN_FEAST_STR.id,
+  SPELLS.BOUNTIFUL_CAPTAIN_FEAST_STA.id,
+];
+
+class FoodChecker extends Analyzer {
+
+  lowerFoodUp = false;
+  higherFoodUp = false;
+
+  on_toPlayer_applybuff(event) {
+    const spellId = event.ability.guid;
+    if (event.prepull) {
+      if(LOWER_FOOD_IDS.includes(spellId)) {
+        this.lowerFoodUp = true;
+      }
+      if(HIGHER_FOOD_IDS.includes(spellId)) {
+        this.lowerFoodUp = true;
+        this.higherFoodUp = true;
+      }
+    }
+  }
+
+  get foodSuggestionThresholds() {
+    return {
+      actual: this.lowerFoodUp,
+      isEqual: false,
+      style: 'boolean',
+    };
+  }
+  get bestFoodSuggestionThresholds() {
+    return {
+      actual: this.higherFoodUp,
+      isEqual: false,
+      style: 'boolean',
+    };
+  }
+
+  suggestions(when) {
+    when(this.foodSuggestionThresholds)
+      .addSuggestion((suggest) => {
+        return suggest(<React.Fragment>You did not had your food up when starting the fight.  Having the right food buff during the combat increase one of your best stat by a fair amount.</React.Fragment>)
+          .icon(SPELLS.KUL_TIRAMISU.icon)
+          .staticImportance(SUGGESTION_IMPORTANCE.MINOR);
+      });
+    when(this.bestFoodSuggestionThresholds)
+      .addSuggestion((suggest) => {
+        return suggest(<React.Fragment>You did not had your best food up when starting the fight.  Having the right food buff during the combat increase one of your best stat by a fair amount.</React.Fragment>)
+          .icon(SPELLS.KUL_TIRAMISU.icon)
```

Comment:
since only the feast is accepted for this, maybe use that as the icon? would also make this more visually distinct from the other suggestion.